src/utilities/validation.tsx

```diff
@@ -58,9 +58,9 @@ export const classValidationRules = {
 
 export const classWorkValidationRules = {
   name: {
-    required: 'Class Type is required',
+    required: 'Class Work Type is required',
```

Comment:
Are we consistently using "class work" as two words or "classwork"? Or is it situational? Just curious, this is probably fine as-is!

Reply:
Mostly it's a placeholder, we may want to switch to more relevant terms like Resources, or Learning/Reading Materials. Open for suggestions.